imblearn/over_sampling/_smote.py

```diff
@@ -1055,6 +1066,11 @@ class KMeansSMOTE(BaseSMOTE):
     --------
     SMOTE : Over-sample using SMOTE.
 
+    SMOTENC : Over-sample using SMOTE for continuous and categorical features.
+
+    SMOTEN : Over-sample using the SMOTE variable specifically for categorical
```

Comment:
SMOTEN : Over-sample using the SMOTE variant specifically for nominal features only.